app/src/services/api.ts

```diff
@@ -0,0 +1,156 @@
+import {emptySplitApi as api} from './emptyApi';
```

Comment:
This looks promising, let's run with this for now. Additionally, we should see how the generator does on a spec that uses all of the features we need around auth and potentially websocket connections to see if we'll need to fill any gaps.